src/main/resources/i18n/messages_fr.properties

```diff
@@ -358,6 +359,8 @@ NavigationSettingsTab.EnableCache=Activer le cache
 NavigationSettingsTab.EnableCacheTooltip=<html><strong>Par d\u00e9faut :</strong> Activ\u00e9<br><strong>AVERTISSEMENT :</strong> D\u00e9sactiver ceci d\u00e9sactivera \u00e9galement la fonctionnalit\u00e9 de suivi des statuts de lecture.<br><strong>Informations :</strong> Quand activ\u00e9, il rend la navigation beaucoup plus rapide et active les fonctionnalit\u00e9s telles que le suivi de statut de lecture.</html>
 NavigationSettingsTab.ResetCache=R\u00e9initialiser le cache
 NavigationSettingsTab.ShowMediaLibraryFolderTooltip=<html><strong>Par d\u00e9faut :</strong> Activ\u00e9<br><strong>Informations :</strong> Si activ\u00e9, la M\u00e9diath\u00e8que sera disponible, ce qui vous permettra de filtrer et de trier dynamiquement vos m\u00e9dias par statut de lecture, genre, notation, etc.</html>
+NetworkTab.MediaServerEngine=Moteur du serveur multim\u00e9dia:
```

Comment:
You might already know, but just making sure, these changes will get overwritten when I sync with Crowdin, so if you want to contribute French translations please do them there

Reply:
It was for my test